Assets/MixedRealityToolkit/Inspectors/Utilities/InspectorFieldsUtility.cs

```diff
@@ -144,6 +186,32 @@ public static void UpdatePropertySettings(SerializedProperty prop, int type, obj
             }
         }
 
+        public static List<InspectorFieldData> GetInspectorFields(System.Object target)
+        {
+            List<InspectorFieldData> fields = new List<InspectorFieldData>();
+            Type myType = target.GetType();
+
+            foreach (PropertyInfo prop in myType.GetProperties())
```

Comment:
btw, I'm not convinced this actually works

Reply:
as in, the code didn't seem to be working for me before the change. I had properties that I wanted to render, and they were not actually getting updated.

Reply:
Yes that bug is exactly #5652 

I added functionality to make it work. The one edge case that is not accounted for is you changing any of the property names will be unrecoverable. There is no FormallySerializedAs in this situation. 

Regarding moving the visual themes to a different folder, no it won't affect customers because the guids are still the same, the classes haven't changed (at least the theme engine ones), the namespaces haven't changed, etc. I don't see how they could be impacted.

![test](https://user-images.githubusercontent.com/25975362/64442682-1438b800-d085-11e9-9c97-cb9235779a07.gif)